File: JAVA/Day5/ParkingSlot.java
/*the clear a variable 
Using the while loop to loop through the method
Use booleen data type
promt the user
using switch case*/






import java.util.Scanner;
import java.util.Array;
public class ParkingSlot {
public static String[] motoPark = int[20];
public static void main(String...args){
Scanner input = new Scanner(System.in);
boolean mainMenu = true;

while(mainMenu){
   String prompt =("""

		  SEMI COLON PARKING SPACE  
			
			choose from the menu:

		1. Park your car	
		2. Remove your car
		3. Show all available parking space
		0. Exist
		""");	
System.out.print(prompt);     	
System.out.print("Enter choice 1, 2, 3 or 0 to quit: ");
int option = input.nextInt();
input.nextLine();

switch(option) {
	case 1: System.out.print("1. Park your car ");
                int userInput = input.nextLine();
                ParkingSlot.addItem(userInput);
                break;
		
	
	case 2: System.out.println("2. Remove items");
                int removeItem = input.nextLine();
                ParkingSlot.removeMyItem(removeItem);
		break;
	
	case 3: System.out.println("3. Show all available parking space");
                String showItem = input.nextLine();
                ParkingSlot.showItem();
		break;
	case 0:
		mainMenu = false;
		System.out.print("Good bye");
		 break;
		
}		
}
}

public static int automobileCar (int numberOfCar){
for(let index = 0; index < carpark.length; index++ )
		System.out.println(itemList);
return "list";
}
public static String removeMyItem(String removeItem){
		itemList.remove(removeItem);
		System.out.println(itemList);
return "list";
}
public static void showItem(){
for(int index = 0; index < itemList.size(); index++){
System.out.println(itemList.get(index));

}
}

}

File: PYTHON/Day5/ParkingSlot.py
#the clear a variable 
#Using the while loop to loop through the method
#Use booleen data type
#promt the user
#using switch case



itemList = [20]
mainMenu = True

while mainMenu:
    menu_text = """
          WELCOME TO SEMICOLON GROCERY STORE  
            
            choose from the menu:

        1. Park your car	
        2. Remove your car
        3. Show all avaible parking space
        0. Exit
    """
    print(menu_text)
    choice = int(input("Enter choice 1, 2, 3 or 0 to quit: "))

    if choice == 1:
        print("1. Park your car: ")
        userInput = input("Enter item: ")
        
	def addMyItem(addItem):
            itemList.append(addItem)
            print(itemList)
            return "list"
        addMyItem(userInput)

    elif choice == 2:
        print("2. Remove your car")
        removeItem = input("Enter item to remove: ")
        def removeMyItem(removeItem):
            if removeItem in itemList:
                itemList.remove(removeItem)
            print(itemList)
            return "list"
        removeMyItem(removeItem)

    elif choice == 3:
        print("3. Show all avaible parking space")
        def showItem():
            for index in range(len(itemList)):
                print(itemList[index])
        showItem()

    elif choice == 0:
        mainMenu = False
        print("Good bye")
